lib/PHPCfg/Parser.php

```diff
@@ -694,8 +700,8 @@ protected function parseExprNode($expr) {
                 // always return the unqualified literal
                 return new Literal($expr->getLast());
             }
-            if (isset($expr->namespacedName)) {
-                return new Literal($expr->namespacedName->toString());
+            if ($expr->isQualified()) {
```

Comment:
The if and else case do the same here. I think this branch was already dead code before, because namespacedName is not put onto Name nodes (rather on things like Class), so this condition was never true.